resources/js/processes/scripts/customFilters.js

```diff
@@ -62,6 +64,7 @@ Vue.filter('lua', function(value) {
       line = line.replace('{dataVariable}',  `data`);
       line = line.replace('{configVariable}',  `config`);
       line = line.replace('{apiExample}',  `users_api:get_users(filter, order_by, order_direction, per_page, include)`);
+      line = line.replace('{apiDocUrl}', `https://github.com/ProcessMaker/sdk-lua#documentation-for-api-endpoints`);
```

Comment:
Missing an S for Docs. Needs to be `apiDocsUrl`

Reply:
Good Catch thank you!